src/azure-cli/azure/cli/command_modules/eventhubs/custom.py

```diff
@@ -305,7 +305,7 @@ def cli_networkrule_createupdate(cmd, client, resource_group_name, namespace_nam
     NWRuleSetIpRules = cmd.get_models('NWRuleSetIpRules', resource_type=ResourceType.MGMT_EVENTHUB)
 
     netwrokruleset = client.get_network_rule_set(resource_group_name, namespace_name)
-
+    logger.warning('This version will be depracated & latest version will release in breaking change release.')
```

Comment:
Please leverage `deprecate_info`

Reply:
Hey @evelyn-ys , I have added the depreciation_info, Can you please check now.

Also for you reference I am attaching PR Link which is in draft, that will show ho will be the future cmdlets for network-rule.
https://github.com/Azure/azure-cli/pull/25792